content/spin/dynamic-configuration.md

```diff
@@ -135,3 +136,43 @@ Got password test_password
 
 Runtime configuration contains information for the selected config provider, such as the [Vault config provider](#vault-config-provider).
 You can supply runtime configuration by providing a value for the `--runtime-config-file` flag when invoking the `spin up` command.
+
+### Key Value Store Runtime Configuration
+
+By default, keys and values will be stored in a SQLite database embedded in Spin. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use an different backing store. Right now, the only store options are the embedded SQLite database or an external Redis database. The following is the default `runtime-config.toml` for the default key-value store.
```

Comment:
time to start a holy war over "a SQLite" vs "an SQLite"

Reply:
on a more serious note: I would strongly encourage us **NOT** to show how to replicate the default setting, as I feel it confuses the issue.  There is no situation in which a user would ever do this, and showing it just makes them think they need to do it.  What we should show is:

* How to change the backing of the `default` store
* How to specify additional stores

Reply:
Ugh, now I am even having self-doubt about saying it is SQLite-backed by default.  Because that's true when you run the Spin binary, but it's _not_ true in Fermyon Cloud.  Maybe we should say something like "what backs the default store is none of your damn business, it just works," and go into detail only when we talk about the runtime config file.